router/core/src/main/scala/io/buoyant/router/ctx.scala

```diff
@@ -0,0 +1,55 @@
+package io.buoyant.router
+
+import com.twitter.finagle._
+import com.twitter.finagle.buoyant.Dst
+import com.twitter.finagle.context.Contexts
+import com.twitter.util.Local
+
+package object ctx {
```

Comment:
what about moving them into a directory AND importing `DstPath` and `DstBound` directly?  Just trying to follow the principle of least surprise.